.devcontainer/post-create.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
-pip3 install schematicpy==24.1.1
+pip3 install schematicpy==24.5.1 ipython==8.18.1
```

Comment:
Will it be better to do: `pip3 install --upgrade schematicpy`, to ensure that the latest version of schematicpy gets installed?

Reply:
LGTM! @linglp , I think we have had better luck pinning the version for our CI workflows, new versions have sometimes had breaking changes that we didn't expect (until the workflow breaks :) )